android-components/components/feature/search/src/test/java/mozilla/components/feature/search/telemetry/ads/AdsTelemetryTest.kt

```diff
@@ -139,7 +139,7 @@ class AdsTelemetryTest {
         assertEquals(Component.FEATURE_SEARCH, facts[0].component)
         assertEquals(Action.INTERACTION, facts[0].action)
         assertEquals(AdsTelemetry.SERP_SHOWN_WITH_ADDS, facts[0].item)
-        assertEquals(telemetry.providerList[0].name, facts[0].value)
+        assertEquals("google.in-content.organic.none", facts[0].value)
```

Comment:
Note that this code is not testing the telemetry (i.e. is not using a Glean testing API and it's not testing that it's recording what we expect it's recording). This is testing that the Fenix abstraction is emitting events in the expected format.